cmd/thv/app/logs.go

```diff
@@ -5,73 +5,90 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/StacklokLabs/toolhive/pkg/container"
 	"github.com/StacklokLabs/toolhive/pkg/labels"
 	"github.com/StacklokLabs/toolhive/pkg/logger"
 )
 
-func newLogsCommand() *cobra.Command {
-	return &cobra.Command{
+var (
+	tailFlag bool
+)
+
+func logsCommand() *cobra.Command {
+	logsCommand := &cobra.Command{
 		Use:   "logs [container-name]",
 		Short: "Output the logs of an MCP server",
 		Long:  `Output the logs of an MCP server managed by Vibe Tool.`,
 		Args:  cobra.ExactArgs(1),
-		Run: func(_ *cobra.Command, args []string) {
-			// Get container name
-			containerName := args[0]
-
-			// Create context
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
-			// Create container runtime
-			runtime, err := container.NewFactory().Create(ctx)
-			if err != nil {
-				logger.Log.Errorf("failed to create container runtime: %v", err)
-				return
-			}
-
-			// List containers to find the one with the given name
-			containers, err := runtime.ListContainers(ctx)
-			if err != nil {
-				logger.Log.Errorf("failed to list containers: %v", err)
-				return
-			}
-
-			// Find the container with the given name
-			var containerID string
-			for _, c := range containers {
-				// Check if the container is managed by Vibe Tool
-				if !labels.IsToolHiveContainer(c.Labels) {
-					continue
-				}
-
-				// Check if the container name matches
-				name := labels.GetContainerName(c.Labels)
-				if name == "" {
-					name = c.Name // Fallback to container name
-				}
-
-				// Check if the name matches (exact match or prefix match)
-				if name == containerName || strings.HasPrefix(c.ID, containerName) {
-					containerID = c.ID
-					break
-				}
-			}
-
-			if containerID == "" {
-				logger.Log.Infof("container %s not found", containerName)
-				return
-			}
-
-			logs, err := runtime.ContainerLogs(ctx, containerID)
-			if err != nil {
-				logger.Log.Errorf("failed to get container logs: %v", err)
-				return
-			}
-			logger.Log.Infof(logs)
-
+		Run: func(cmd *cobra.Command, args []string) {
+			logsCmdFunc(cmd, args)
```

Comment:
Should we use RunE instead? I'm not keen on not returning errors

Reply:
Yep, I felt the same way tbh. Although I kind of put it in the back of my mind todo later on when we refactor a bunch of the subcommand files (removing the `init()` functions, and making them all consistent).
I'll add it for the logs case just for a reference for myself (or others) later on when we refactor